src/commands/index.ts

```diff
@@ -151,6 +151,14 @@ export function load(client: any): any {
     client.internaltesting.functions = {};
     client.internaltesting.functions.discover = loadCommand("internaltesting-functions-discover");
   }
+  if (experiments.isEnabled("frameworks")) {
```

Comment:
we are changing the name of the experiments based on user feedback:

https://github.com/firebase/firebase-tools/pull/6439